<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>交易管理系统(交易员)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget_strategy" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget_strategy"/>
       </item>
       <item>
        <widget class="QPushButton" name="pb_start">
         <property name="text">
          <string>启动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_stop">
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_log" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>日志信息：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_log">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pb_dir">
            <property name="text">
             <string>选择文件夹</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_file">
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_strategyManager">
    <property name="title">
     <string>策略管理</string>
    </property>
    <addaction name="action_addStragety"/>
    <addaction name="action_deleteStragety"/>
   </widget>
   <widget class="QMenu" name="menu_log">
    <property name="title">
     <string>日志与监控</string>
    </property>
    <addaction name="action_addRisk"/>
    <addaction name="action_deleteRisk"/>
   </widget>
   <addaction name="menu_strategyManager"/>
   <addaction name="menu_log"/>
  </widget>
  <action name="actionxx">
   <property name="text">
    <string>添加资金账户</string>
   </property>
  </action>
  <action name="actionbb">
   <property name="text">
    <string>删除资金账户</string>
   </property>
  </action>
  <action name="action_addStragety">
   <property name="text">
    <string>添加策略</string>
   </property>
  </action>
  <action name="action_deleteStragety">
   <property name="text">
    <string>删除策略</string>
   </property>
  </action>
  <action name="action_addRisk">
   <property name="text">
    <string>添加风控指标</string>
   </property>
  </action>
  <action name="action_deleteRisk">
   <property name="text">
    <string>删除风控指标</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
